src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -28,6 +31,7 @@ object SharedFileSystem {
     case "copy" => localizePathViaCopy _
     case unsupported => throw new UnsupportedOperationException(s"Localization strategy $unsupported is not recognized")
   })
+  val CallLocalizers = Stream(localizePathAlreadyLocalized _, localizePathViaSymbolicLink _, localizePathViaHardLink _, localizePathViaCopy _)
```

Comment:
Yes I would definitely suggest linking first for efficiency reasons.  But it would be nice to be able to force copy-only via only config changes.  Which all happens to align with the shared filesystem localization strategy that's already in place for workflow inputs.  :smile: